PowerShell/JumpCloud Module/Docs/Get-JCSystemApp.md

```diff
@@ -0,0 +1,159 @@
+---
+external help file: JumpCloud-help.xml
+Module Name: JumpCloud
+online version: https://github.com/TheJumpCloud/support/wiki/
+schema: 2.0.0
+---
+
+# Get-JCSystemApp
+
+## SYNOPSIS
+Returns the System Apps installed to a JumpCloud System(s)
+
+## SYNTAX
+
+### All (Default)
+```
+Get-JCSystemApp [-SystemID <String>] [-SystemOS <String>] [-SoftwareName <String>] [-SoftwareVersion <String>]
+ [<CommonParameters>]
+```
+
+### Search
+```
+Get-JCSystemApp [-SystemID <String>] [-SystemOS <String>] [-SoftwareName <String>] [-SoftwareVersion <String>]
+ [-Search] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get-JCSystem app function enables admins to search what software is on their systems
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-JCSystemApp -SystemId 'SystemId'
+```
+
+Returns the applications installed in the system with the given -SystemId
+
+### Example 2
+```powershell
+PS C:\> Get-JCSystemApp -SystemOs 'MacOs'
+```
+
+Returns the 'MacOs' systems and all the applications installed for each system
+
+### Example 3
+```powershell
+PS C:\> Get-JCSystemApp -SystemOs 'MacOs' -SoftwareName 'JumpCloud-Agent'
+```
+
+### Example 4
+```powershell
+PS C:\> Get-JCSystemApp -SystemOs 'MacOs' -SoftwareName 'JumpCloud-Agent' -SoftwareVersion '1.12.5'
+```
+
+Returns the 'MacOs' systems that have a 'JumpCloud Agent' application with the version '1.12.5'
+
+### Example 5
+```powershell
+PS C:\> Get-JCSystemApp -SoftwareName 'jumpcloud-agent' -Search
+```
+
+Returns any 'jumpcloud-agent' software installed in all the os systems
+
```

Comment:
I think we should also call out search by os and id examples here. Another example which would be immedietly helpful for folks would be:

Get-JCSoftwareApp -SystemOS "windows" 

Which would just return all windows programs